manager.py

```diff
@@ -125,7 +125,9 @@ def __init__(self,
             verbosity=self.v,
             logfile=self.logfile)
         # DEFAULT_UDP_PORT and DEFAULT_TCP_PORT are assigned in sender
-
+        self.branch = 'master'
```

Comment:
Hmm, this could potentially cause undesired behavior if the station is on a different branch, `Manager` gets run, no TCP packets are returned so `self.branch` doesn't get set to what it's supposed to be, and then a reboot happens. It would reboot onto `master` instead of the current branch.

Instead, let's set `self.branch = ''` and I'll have `git-pull-reboot.sh` check for an empty `branch` argument and stay on the current branch in that case.